samples/cpp/benchmark_app/utils.cpp

```diff
@@ -662,65 +662,7 @@ std::vector<benchmark_app::InputsInfo> get_inputs_info(const std::string& shape_
                            reshape_required);
 }
 
-#ifdef USE_OPENCV
```

Comment:
There are other places also use macro `USE_OPENCV`, do we need consider removing them completely if OpenCV will not used any more?

Reply:
> There are other places also use macro `USE_OPENCV`, do we need consider removing them completely if OpenCV will not used any more?

OpenCV is still used for reading of different image formats. Without OpenCV bmp is supported only